play-media-native/api/queries/getMedia/index.ts

```diff
@@ -14,7 +16,13 @@ query {
       fileHeight,
       fileSize,
       fileType,
-      fileUrl,
+      fileUrl (
+        transform: {
+          width: ${Math.ceil(Dimensions.get('window').width)},
+          height: ${Math.ceil(Dimensions.get('window').height / 2)},
```

Comment:
Why half of the height? What if the image is vertical?

Reply:
Even if the image is vertical it still can't take up full device height in grid, list or card view. For the getMediaById query it is fetched with full device height transformation.

Reply:
Oh, makes sense!